cmd/buildah/from.go

```diff
@@ -36,6 +37,11 @@ var (
 			Value: DefaultRegistry,
 		},
 		cli.StringFlag{
+			Name:  "creds",
+			Value: "",
+			Usage: "Use `USERNAME[:PASSWORD]` for accessing the registry",
```

Comment:
@runcom 

Line 42 is also 

Usage: "use `username[:password]` for accessing the registry",